src/ice_usb.c

```diff
@@ -57,10 +64,18 @@ void tud_dfu_download_cb(uint8_t alt, uint16_t block_num, const uint8_t *data, u
 // Invoked when download process is complete, received DFU_DNLOAD (wLength=0) following by DFU_GETSTATUS (state=Manifest)
 // Application can do checksum, or actual flashing if buffered entire image previously.
 // Once finished flashing, application must call tud_dfu_finish_flashing()
-void tud_dfu_manifest_cb(uint8_t alt)
-{
-    tud_dfu_finish_flashing(DFU_STATUS_OK);
-    watchdog_reboot(0, 0, 1000);
+void tud_dfu_manifest_cb(uint8_t alt) {
+    assert(dfu_download_pending);
+    dfu_download_pending = false;
+
+    ice_fpga_flash_deinit();
+    bool fpga_done = ice_fpga_reset();
```

Comment:
That makes sense to leave all the board's chip in a reset state when the MCU is rebooting.